### PR TITLE
[pfedit] Fix master build due to merge conflict

### PR DESCRIPTION
**Pull request by ejgallego**

### CODE REVIEW

**Comment by ejgallego** [comment]
This needs review cc @SkySkimmer

File: proofs/pfedit.ml [hunk 1]
```diff
@@ -173,7 +173,7 @@ let build_by_tactic ?(side_eff=true) env sigma ?(poly=false) typ tac =
       const_entry_body = Future.chain ce.const_entry_body
         (fun (pt, _) -> pt, ()) } in
   let (cb, ctx), () = Future.force ce.const_entry_body in
-  let univs = UState.merge side_eff Evd.univ_rigid univs ctx in
+  let univs = UState.merge ~sideff:side_eff ~extend:status Evd.univ_rigid univs ctx in
```
Comment (SkySkimmer) [inline proofs/pfedit.ml]: I don't think that's the right value for `extend`, it should probably be `true`.
Isn't the `status` about using `admit`?

**Review by SkySkimmer — CHANGES_REQUESTED** [review]

File: proofs/pfedit.ml (hunk 1 above)
Comment (ejgallego) [reply proofs/pfedit.ml]: I have no idea, setting to true for now.

**Review by SkySkimmer — APPROVED** [review]

**Comment by SkySkimmer** [comment]
I'm going to merge since otherwise everything is broken + this is what was intended in #8302.

**Pull request merged**